docs/aws/aws.md

```diff
@@ -196,3 +196,24 @@ See the egress golang examples above, and replace the line starting with `out :=
 ```go
 out := cli.VerifyDns(context.TODO(), "vpcID")
 ```
+
+## Interface ##
```

Comment:
I think defining this in the main `README` or in something like `docs/platform.md` once would be easier and less duplication.

Reply:
Agreed, moving to main README